commands/policy.go

```diff
@@ -23,8 +46,9 @@ func NewPolicy(network *net.Network) (cmd *Policy) {
 func (cmd *Policy) Metadata() command_metadata.CommandMetadata {
 	return command_metadata.CommandMetadata{
 		Name:        "policy",
-		Description: "Show the status of a policy for an application and entity",
-		Usage:       "BROOKLYN_NAME SCOPE policy POLICY",
+        Aliases:     []string{"policies","pol","pols"},
```

Comment:
"policy", not "policies", for consistency with other commands and the usage below.